docs/key-issues/gpus.md

```diff
@@ -30,54 +30,78 @@ Packaging such projects for PyPI has been, and still is, quite challenging.
 
 ## Current state
 
-As of December 2022, PyPI and Python packaging tools are completely unaware of
+As of May 2023, PyPI and Python packaging tools are completely unaware of
```

Comment:
Months pass quickly (hey, it's June already!). How about the "As of mid 2023"?

Reply:
Good call, updated.